### ISSUE TITLE
Invoice Created

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#340]
In GitLab by @kdjstudios on May 12, 2017, 08:31

Helpdesk: http://www.servicedesk.answernet.com/profiles/ticket/2017-05-12-39127/conversation

User dtiwari with email address pchaudhary@tekege.com created the invoice of College Station Medical Associates , account number CSM010 of Laser Answering Service at 12/May/2017, 09:46 AM EDT

Thank You!

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 12, 2017, 08:33

@dtiwari @smasih @pchaudhary 

This appears to be a test notification for manually created invoices. Is this correct?

Also idk if this is a test email or a real email: <atrussell@laseranswering.com> but we should be sure to be using test accounts when testing, to insure actually clients are not sent testing info.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on May 12, 2017, 08:55

@kdjstudios   
Hi,  
Yes this is a test notification for manually created invoices  
and also **atrussell@laseranswering.com** is a real email address of site **Laser Answering Service** but we used this address as a sender only not recepient.  
But when we  sent an email to **support@sabilling.com**, we got to know that the copy of the same we received is also sent to **atrussell@laseranswering.com**.  
  
  
````

Your request has been received by SA Billing Support Team.

      Subject:   Invoice Created
Ticket Number:   2017-05-12-39127

For all matters related to this ticket please respond to this email.  A member of the Support team will contact you shortly to resolve your issue.  
````  
  
  
We apologize for the incovinience caused by the accident because we have no idea that the **support@sabilling.com** is having the functionality to send the  email to all the  marked users.  
We request you to update the same to **atrussell@laseranswering.com**.  
We will take care of this from next time.
